app/bundles/PageBundle/Controller/PublicController.php

```diff
@@ -463,9 +464,10 @@ public function redirectAction(
         // Get query string
         $query = $request->query->all();
 
-        // Unset the clickthrough from the URL query
-        $ct = $query['ct'] ?? null;
-        unset($query['ct']);
+        // @aivie Tracking: The CT is added to the path of the URL. Not as parameter.
+        $clickthrough = $redirectModel->decodeArrayFromUrl($ct);
```

Comment:
optionally fetch the ct from GET.

Reply:
I agree with @galvani. Are you OK that all existing redirects will start breaking? All the emails that you've already sent to your contacts will have broken links. Can you also explain the comment? I need some help understanding why it's there.

Reply:
@galvani @escopecz Thanks! You are right. I missed this because the major version.  As far as the PR is closed it doesn't make sense to fix it. I'll be happy to test the PR @kuzmany.